library/Faktory/Ent/Batch/Status.hs

```diff
@@ -0,0 +1,48 @@
+module Faktory.Ent.Batch.Status
+  ( jobBatchId
+  , BatchStatus(..)
+  , batchStatus
+  ) where
+
+import Faktory.Prelude
+
+import Control.Error.Util (hush)
+import Data.Aeson
+import Data.ByteString.Lazy as BSL
+import Data.Text.Encoding (encodeUtf8)
+import Data.Time (UTCTime)
+import Faktory.Client
+import Faktory.Ent.Batch
+import Faktory.Job (Job, jobOptions)
+import Faktory.Job.Custom
+import Faktory.JobOptions (JobOptions(..))
+import Faktory.Producer
+import GHC.Generics
+
+data BatchStatus = BatchStatus
+  { bid :: BatchId
+  , total :: Int
+  , pending :: Int
+  , failed :: Int
+  , created_at :: UTCTime
+  , description :: Text
+  }
+  deriving stock Generic
+  deriving anyclass FromJSON
+
+newtype ReadCustomBatchId = ReadCustomBatchId
+  { _bid :: BatchId
```

Comment:
Really love that you enqueue things with `{ custom: { bid: x } }` and you get it back as `{ custom: { _bid: x } } `, super great :ok_hand: